pkg/cmd/pulumi/policy_ls.go

```diff
@@ -51,31 +53,44 @@ func newPolicyLsCmd() *cobra.Command {
 				}
 			}
 
-			// List the Policy Packs for the organization.
+			// Gather all Policy Packs for the organization.
 			ctx := context.Background()
-			policyPacks, err := b.ListPolicyPacks(ctx, orgName)
-			if err != nil {
-				return err
+			var (
+				allPolicyPacks []apitype.PolicyPackWithVersions
+				inContToken    backend.ContinuationToken
+			)
+			for {
+				resp, outContToken, err := b.ListPolicyPacks(ctx, orgName, inContToken)
+				if err != nil {
+					return err
+				}
+
+				allPolicyPacks = append(allPolicyPacks, resp.PolicyPacks...)
+
+				if outContToken == nil {
+					break
+				}
+				inContToken = outContToken
 			}
```

Comment:
feels like it _might_ be worth writing a little helper to encapsulate the token pattern, e.g.

```go
func listWithContinuation(list func(token backend.ContinuationToken) (backend.ContinuationToken, error) (err error) {
	var token backend.ContinuationToken
	for {
		token, err = list(token)
		if err != nil {
			return err
		}
		if token == nil {
			return nil
		}
	}
}
```

Then callers could be something like 

```go
var allPolicyPacks []apitype.PolicyPackWithVersions
if err := listWithContinuation(func (token backend.ContinuationToken) (backend.ContinuationToken, error) {
	resp, outContToken, err := b.ListPolicyPacks(ctx, orgName, inContToken)
	allPolicyPacks = append(allPolicyPacks, resp.PolicyPacks...)
	return outContToken, err
}); err != nil {
	return err
}
```

Honestly, though, it's a pretty simple pattern and I don't feel strongly about this.

Reply:
That's a reasonable suggestion, but I'm inclined to keep this as-is. We only would use this in 3x places, and one of those will 🤞 be replaced with a much slicker "output stacks while we receive them" rather than batching up all the results synchronously before proceeding.